tests/unit_tests/test_swap_by_newest.py

```diff
@@ -0,0 +1,105 @@
+import pytest
+from swap_meet.vendor import Vendor
+from swap_meet.item import Item
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
+
+def test_swap_by_newest_nominal_case():
```

Comment:
Small tweak here, but `nominal_case` doesn't necessarily give someone viewing this test to go off of! You could just leave it at `test_swap_by_newest` and you would be good to go!